selftests/unit/test_config_change_validation.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import os
+import unittest
+
+#from avocado.core import exceptions
```

Comment:
Bah, forgot to remove this.  Will do on next version.

Reply:
Fair enough - the code looks fine and matches what I have migrated as far as I can say. As I am not capable of reviews and this code matches my original proposal, I leave the rest of approving the PR to the others.